substrate/client/src/in_mem.rs

```diff
@@ -26,7 +26,8 @@ use primitives::block::{self, Id as BlockId, HeaderHash};
 use blockchain::{self, BlockStatus};
 use state_machine::backend::{Backend as StateBackend, InMemory};
 
-fn header_hash(header: &block::Header) -> block::HeaderHash {
+/// Compute block header hash.
+pub fn header_hash(header: &block::Header) -> block::HeaderHash {
```

Comment:
this method is redundant with `blake2_256()`, isn't it?

Reply:
thanks, replaced with direct call